jmeter-plugins/src/main/resources/messages.properties

```diff
@@ -0,0 +1,2 @@
+x_correlation_id_prefix=A custom prefix for X-Correlation-ID
+function_name_paropt=Name of variable in which to store the result (optional)
```

Comment:
what is a "function_name_paropt"? I think this would be clearer as "variable_name", or something else more descriptive.

Reply:
The "function_name_paropt" is used throughout the JMeter function implementation, since it is already defined as a resource. So I keep the name consistent with JMeter way of naming the var. It's am awkward name I agree but I prefer to stick with the convention.